lib/gateway/service-map.js

```diff
@@ -85,7 +85,8 @@ async function getRemoteSchemaDefinition (serviceConfig, initHeaders) {
         }
         `
     }),
-    headers
+    headers,
+    context: {} // No context to pass at this stage
```

Comment:
if there's no context, do we need to provide a fake one then or would leaving it undefined be a more sensible default than providing an empty object then?